worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/listener/BrushListener.java

```diff
@@ -1,8 +1,8 @@
 package com.fastasyncworldedit.bukkit.listener;
 
-import com.fastasyncworldedit.core.object.brush.MovableTool;
-import com.fastasyncworldedit.core.object.brush.ResettableTool;
-import com.fastasyncworldedit.core.object.brush.scroll.ScrollTool;
+import com.fastasyncworldedit.core.command.tool.brush.MovableTool;
+import com.fastasyncworldedit.core.command.tool.brush.ResettableTool;
+import com.fastasyncworldedit.core.command.tool.brush.scroll.ScrollTool;
```

Comment:
Why not just `command.tool`? WorldEdit doesn't use `command.tool.brush` for XXXXTool